lib/src/flutter_deck_configuration.dart

```diff
@@ -84,24 +90,28 @@ class FlutterDeckSlideConfiguration extends FlutterDeckConfiguration {
     FlutterDeckHeaderConfiguration? header,
     bool? showProgress,
     FlutterDeckTransition? transition,
+    FlutterDeckProgressIndicator? progressIndicator,
   })  : _footerConfigurationOverride = footer,
         _headerConfigurationOverride = header,
         _showProgressOverride = showProgress,
+        _progressIndicatorOverride = progressIndicator,
         _transitionOverride = transition;
 
   /// Creates a configuration for a slide. This constructor is used internally
   /// to create a configuration when the global configuration is overridden.
-  const FlutterDeckSlideConfiguration._({
-    required this.route,
-    this.hidden = false,
-    this.steps = 1,
-    super.footer,
-    super.header,
-    super.showProgress,
-    super.transition,
-  })  : _footerConfigurationOverride = null,
+  const FlutterDeckSlideConfiguration._(
+      {required this.route,
```

Comment:
Some strang formatting over there:
```suggestion
  const FlutterDeckSlideConfiguration._({
    required this.route,
```